data/piyush-sri.json

```diff
@@ -10,7 +10,8 @@
       "url": "https://www.linkedin.com/in/piyush-srivastava-395841192/",
       "icon": "FaLinkedin"
     },
-    {
+     {
```

Comment:
I think the previous one was correct. 🤔

Reply:
formatting will be fixed by the GitHub Action after a pull request is merged